### ISSUE TITLE
Cross compile armv6hf fails with swing errors

### ISSUE THREAD

**Issue by dellgreen** [javafxports/openjdk-jfx#54]
Swing shouldn't be building by default.
Un-commenting the following from build.gradle line 2154 allows the build to succeed

/* should not be built, but needed in JMX 
    tasks.all { 
        if (!COMPILE_SWING) it.enabled = false 
    } 
   */ 

[JDK-8195669](https://bugs.openjdk.java.net/browse/JDK-8195669?jql=text%20~%20%22javafx%20arm%22%20ORDER%20BY%20created%20DESC)

http://hg.openjdk.java.net/openjfx/jfx-dev/rt/

**Comment by imario42** [comment]
Here you can see what I did to make it compile for arm

https://github.com/javafxports/openjdk-jfx/compare/master...imario42:master

**Comment by dellgreen** [comment]
cool thanks

**Comment by kevinrushforth** [comment]
At first glance, this looks fine, although please also remove the following comment, since you will have addressed it:

```
     // Now that JMX is gone, we can likely uncomment the following as
     // a fix for JDK-8195669
     /* should not be built, but needed in JMX
```

Also, since this touches shared build logic I will do a test build once you provide the PR.
